src/commands/convert.ts

```diff
@@ -0,0 +1,56 @@
+import { promises as fPromises } from 'fs';
+import { flags } from '@oclif/command';
+import Command from '../base';
+import { ValidationError } from '../errors/validation-error';
+import { load } from '../models/SpecificationFile';
+import { SpecificationFileNotFound } from '../errors/specification-file';
+import { convert } from '@asyncapi/converter';
+import specs from '@asyncapi/specs';
+
+const latestVersion = Object.keys(specs).pop() as string;
+
+export default class Convert extends Command {
+  static description = 'convert asyncapi documents older to newer versions';
+
+  static flags = {
+    help: flags.help({ char: 'h' }),
+    output: flags.string({ char: 'o', description: 'path to the file where the result is saved' }),
+    'target-version': flags.string({ char: 't', description: 'asyncapi version to convert to', default: latestVersion })
+  }
```

Comment:
Please add `--watch, -w` flag like here https://github.com/asyncapi/cli/blob/master/src/commands/validate.ts#L14 and update `run` function like here https://github.com/asyncapi/cli/blob/master/src/commands/validate.ts#L28 to support watching of file :)

Reply:
🤔 Do you think it makes sense to have `--watch` on the convert command? I can't imagine anyone editing an old AsyncAPI file while converting. Am I missing something?

Reply:
Right, I don't remember what I was thinking when I wrote that comment that the flag `watch` here would be needed 😅 

@peter-rr Please don't integrate `--watch` flag. Here it's unnecessary.